app/build.gradle

```diff
@@ -49,6 +50,7 @@ dependencies {
 
     // This version is compatible with current spring version: 5.3.8
     testImplementation "org.apache.camel:camel-test-spring-junit5:${camel_version}"
+    testImplementation "org.slf4j:slf4j-simple:2.0.3"
```

Comment:
Why do we need this? I thought this was default in lombok.

Reply:
This is the simple implementation for log4j. I added it because I want to see the logs when I run tests. Without it, we don't get logging